drivers/fastboot/fb_command.c

```diff
@@ -86,6 +86,15 @@ static const struct {
 		.dispatch = oem_format,
```

Comment:
Hi,

First, regarding the commit title and log, please follow the linux kernel rules:
https://github.com/torvalds/linux/blob/master/Documentation/process/submitting-patches.rst#11-sign-your-work---the-developers-certificate-of-origin
https://github.com/torvalds/linux/blob/master/Documentation/process/submitting-patches.rst#14-the-canonical-patch-format

Thanks!